src/frida/tracer.py

```diff
@@ -308,6 +309,30 @@ def _notify_update(self, function, handler, source):
             self._on_update_callback(function, handler, source)
 
     def _create_stub_handler(self, function):
+        args = ""
+        argc = 0
+        varargs = False
+        try:
+            output = subprocess.check_output(["man", "-P", "col -b", "2", function.name], stderr=subprocess.DEVNULL)
```

Comment:
No, because `-b` is an option to `col`, not `man`.
The `-P` option specifies the pager, in this case `col -b`, which removes the escape codes from the output making it plaintext.